### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by Matp101**
The setup files for getting docker working with run and build scripts

### CODE REVIEW

**Comment by zainaraza43** [comment]
not now, we're trying to do things !!!

**Comment by Matp101** [comment]
Man, so sad :(

File: Dockerfile [hunk 1]
```diff
@@ -13,14 +13,8 @@ COPY requirements.txt /app/
 # Install the dependencies
 RUN pip install --no-cache-dir -r requirements.txt
```
Comment (JagrajAulakh) [inline Dockerfile]: We need to change this to use poetry

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Flask==2.3.2
+Flask-SQLAlchemy==3.0.5
+Flask-Login==0.6.3
+python-dotenv==1.0.0
+bcrypt==4.0.1
```
Comment (JagrajAulakh) [inline requirements.txt]: NO REQUIREMENTS!

File: Dockerfile (hunk 1 above)
Comment (zainaraza43) [reply Dockerfile]: Take a look at [how I did Dockerfile with poetry](https://github.com/zainaraza43/the-house/blob/main/Dockerfile)

**Review by JagrajAulakh — APPROVED** [review]

**Review by zainaraza43 — APPROVED** [review]

**Pull request merged**